src/rules/database-migration-filename-format.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Module dependencies.
+ */
+
+import dayjs from 'dayjs';
+import dayjsCustomParseFormat from 'dayjs/plugin/customParseFormat.js';
+import dayjsUtcPlugin from 'dayjs/plugin/utc.js';
+import path from 'node:path';
+
+/**
+ * Enable support for custom parse format and parse and display UTC to Day.js.
+ */
+
+dayjs.extend(dayjsCustomParseFormat);
+dayjs.extend(dayjsUtcPlugin);
+
+/**
+ * Constants.
+ */
+
+const FILENAME_REGEX = /^([0-9]{14})-([a-z-]*)$/;
```

Comment:
```suggestion
const FILENAME_REGEX = /^([0-9]{14})-([a-z-]+)$/;
```
I don't think we should allow empty migration names.
Should we have a minimum of length?

Reply:
Not sure also about the minimum length 🤔